### ISSUE TITLE
Why update_lr() ?

### ISSUE THREAD

**Issue by oneHuster** [github-pengge/PyTorch-progressive_growing_of_gans#14]
I was so confused about using `self.update_lr(cur_nimg)`  in train.py [train.py#L281](https://github.com/github-pengge/PyTorch-progressive_growing_of_gans/blob/8337fc9b536f0db402be9fa40f791afaa9be7855/train.py#L281) which may produce negative learning rate coefficient during the training process.

I have no idea about why we should use this func --- a func just make learning rate fluctuating.

**Comment by github-pengge** [comment]
This is a trick taken from official code.
